static/src/js/components/FormNavigation/__tests__/FormNavigation.test.jsx

```diff
@@ -10,6 +10,7 @@ import {
   Route,
   Routes
 } from 'react-router'
+import { beforeEach } from 'vitest'
```

Comment:
This import shouldn't be needed. The `globals: true` config in `vite.config.js` (in the test object) means we don't have to import from `vitest` in every test